packages/babel-helpers/src/helpers.js

```diff
@@ -372,6 +372,29 @@ helpers.extends = defineHelper(`
   }
 `);
 
+helpers.extendsDefine = defineHelper(`
+  export default function _extendsDefine() {
+    _extendsDefine = function (target) {
+      for (var i = 1; i < arguments.length; i++) {
+        var source = arguments[i];
+        Object.getOwnPropertyNames(source).forEach(function(propKey) {
+          var desc = Object.getOwnPropertyDescriptor(source, propKey);
+          Object.defineProperty(target, propKey, desc);
+        });
+        if (Object.getOwnPropertySymbols) {
```

Comment:
Could we have a more explicit check here:

```js
typeof Object.getOwnPropertySymbols === 'function'
```

Reply:
@xtuc 👌  for me… I chose this one because it's actually being used in other helpers such as `objectWithoutProperties` and `defineEnumerableProperties`. Would you like me to change those as well?

Reply:
We can change them in a later pr, it's not a big deal actually but we need it's better that our helper are fast enough.

Reply:
@xtuc hi! just added a new commit for fixing all the code review comment... this check has been removed now...